.pipelines/.vsts-vhd-builder-release.yaml

```diff
@@ -13,75 +41,75 @@ parameters:
   - name: build1804containerd
     displayName: Build 1804 containerd
     type: boolean
-    default: true
+    default: false
```

Comment:
we rely on default values here to know which SKUs to build during official releases

Reply:
why is there SKUs we don't release?